repos/system_upgrade/el7toel8/actors/vimcheck/actor.py

```diff
@@ -0,0 +1,32 @@
+from leapp.actors import Actor
+from leapp.libraries.common.rpms import has_package
+from leapp.models import InstalledRedHatSignedRPM, Report
+from leapp import reporting
+from leapp.reporting import create_report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class VimCheck(Actor):
+    """
+    Check if vim-minimal or vim-enhanced is installed and if so, inform about planned actions.
+    """
+
+    name = 'vim_check'
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (Report,)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        vim_pkgs = ['vim-minimal', 'vim-enhanced']
+        if any(has_package(InstalledRedHatSignedRPM, pkg) for pkg in vim_pkgs):
+            # NOTE: informational message about planned fixes during IPU
+            create_report([
```

Comment:
My POV: I would not create a report message about this info at all. I feel like it does bring little value to the user, because even if they don't get to know about it, the behaviour would just work as before. Basically, we strive for a concise pre-upgrade report so that users don't get discouraged by a ton of messages to read.

@shaded-enmity POV: If you want to give the user information about Leapp changing their vim configuration, do that **after** it really changes, that means in the VimMigrate actor.

If you choose the latter, please use the reporting.Severity.INFO instead of LOW.
Also, at this time, the report message won't be collected and printed in any report, but we plan to change that in the future. We'll be collecting the report messages produced after the pre-upgrade report and generate the post-upgrade report.